wlauto/config_example.py

```diff
@@ -286,4 +286,8 @@
 # Scratch resource getter.
 #scratch_mount_point = '/arm/scratch'
 
+# URL of server from where to fetch external resources for workloads on-demand.
+# Used by WA's get-assets sub-command.
+remote_assets_url = 'https://github.com/jimboatarm/wa-assets/raw/master/dependencies'
```

Comment:
Yes, that priority is correct.

On 04 Aug 2016, 05:20:15 -0700, muendelezaji wrote:
> > @@ -286,4 +286,8 @@
> >  # Scratch resource getter.
> >  #scratch_mount_point = '/arm/scratch'
> >
> > +# URL of server from where to fetch external resources for workloads on-demand.
> > +# Used by WA's get-assets sub-command.
> > +remote_assets_url = 'https://github.com/jimboatarm/wa-assets/raw/master/dependencies'
>
> So the order of priority would be:
> 1. `--url` argument if specified
> 2. `remote_assets_url` config if present
> 3. default value inside `get-assets` script
>
> Does that look OK?
>
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/ARM-software/workload-automation/pull/219/files/80193284aecba6b90b881e165f6b51ffb448ea00#r73511065
IMPORTANT NOTICE: The contents of this email and any attachments are confidential and may also be privileged. If you are not the intended recipient, please notify the sender immediately and do not disclose the contents to any other person, use it for any purpose, or store or copy the information in any medium. Thank you.